### PR TITLE
VACMS-4284: Uninstall OpenAPI, OpenAPI UI, ReDoc for OpenAPI UI, and Swagger UI for OpenAPI UI

### PR DESCRIPTION
**Pull request by indytechcook**
## Description

See #4284. 

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [x] `Core Application Team`
- [ ] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [x] No announcement is needed for this code change. 
  - [x] Merge & carry on unburdened by announcements

---
This change is [Reviewable](https://reviewable.io/reviews/department-of-veterans-affairs/va.gov-cms/4716)

### CODE REVIEW

File: READMES/devops/deploy-process.md [hunk 1]
```diff
@@ -4,27 +4,21 @@ The purpose of this table is to show the steps the deploy process follows and wh
...
+Notification in Slack | Current | Jenkins | Yes
+Wait 60 minutes | Current | Jenkins | Yes
+Get latest passing commit | Current | Jenkins
+Find AMI for latest passing commit | Current | Jenkins | Yes
+Enable Site Alert | Current | Current | No
+Activate "Maintenance Mode" | Current | Current | No
+Back up files to S3 | Current | Current | No
+Add new AMI to load balancer | Current | Jenkins | No
+New server comes online but not not live | Current | New | No
+Run Drush Deploy Command (https://www.drush.org/latest/deploycommand/) | Current | New | No
```
Comment (acrollet) [inline READMES/devops/deploy-process.md:19]:
```suggestion
Run [Drush Deploy Command](https://www.drush.org/latest/deploycommand/) | Current | New | No
```

File: docroot/modules/custom/va_gov_build_trigger/src/SiteStatus/SiteStatusInterface.php [hunk 2]
```diff
@@ -15,7 +15,7 @@ interface SiteStatusInterface {
    * occur to an end user.
    *
    * Deploy mode is disabled when the all code deploy, config import,
-   * database update and cms bulk cms export have been completed.
+   * and database have been completed.
```
Comment (acrollet) [inline docroot/modules/custom/va_gov_build_trigger/src/SiteStatus/SiteStatusInterface.php:18]:
```suggestion
   * and database updates have been completed.
```

**Review by acrollet — COMMENTED** [review]
This is looking good, I think the testing failure is a flicker. I'm merging master and also removing the `Drupal\va_gov_content_export` in `composer.json`. Not seeing any other leftover references... Added a couple very minor textual suggestions.

**Comment by acrollet** [comment]
> I'm merging master and also removing the `Drupal\va_gov_content_export` in `composer.json`.

whoops, no push permission to your fork - I created a PR.

**Review by indytechcook — COMMENTED** [review]
[DOCS] Better words

**Comment by indytechcook** [comment]
> I'm merging master and also removing the Drupal\va_gov_content_export in composer.json.

I removed the files in the branch

**Comment by indytechcook** [comment]
@acrollet i resolved the conflict but think I lost your text suggestions.

**Comment by acrollet** [comment]
> @acrollet i resolved the conflict but think I lost your text suggestions.

I'm still seeing them :-)

**Review by acrollet — APPROVED** [review]

**Pull request merged**